lib/apr/views/commerce_slack_view.ex

```diff
@@ -21,6 +22,9 @@ defmodule Apr.Views.CommerceSlackView do
       routing_key =~ "order." ->
         CommerceOrderSlackView.render(subscription, event, routing_key)
 
+      routing_key == "disqualified" -> 
```

Comment:
Will the whole routing key be "disqualified" or will it just contain "disqualified"? (The [Artsy event service](https://github.com/artsy/artsy-eventservice/blob/master/lib/artsy-eventservice/presenters/events/base_event.rb#L47-L49) makes me think it will be "shippingquoterequestevent.disqualified".)

Reply:
Great catch, thank you!
Just verified by checking an object in Exchange and it's really set to `shippingquoterequestevent.disqualified`. Pushed these changes in d44a568998495bf181ab72b95e0b645d38a41c5a

Reply:
Great catch Adam! and thanks for adjusting @lidimayra !